packages/syntax-icons/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@cambly/syntax-icons",
+  "description": "Cambly design system Icon Library",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "sideEffects": false,
+  "license": "Apache-2.0",
+  "files": [
+    "dist/**"
+  ],
+  "scripts": {
+    "build": "tsup src/index.tsx --format esm,cjs --dts --external react",
```

Comment:
@somethiiing I don't think this is correct - don't we also need to build the icons themselves?

Reply:
@somethiiing something along these lines (didn't test):
```
    "build": "tsup src/*.tsx --format esm,cjs --dts --external react",
    "dev": "tsup src/*.tsx --format esm,cjs --watch --dts --external react",
```